tap_workday_raas/discover.py

```diff
@@ -25,7 +29,7 @@ def _element_to_schema(element):
         #TODO What else do we need for decimal types?
         schema = {
             'type': ['number'],
-            'format': 'decimal'
+            'format': 'singer-decimal'
```

Comment:
For singer-decimal, the type must be `string`, and we should confirm that the Transformer allows it.

Given that folks are using it, I think changing this should be reserved for a major version bump. Having them properly typed as `Decimal`s will cause inconsistent types.

TLDR - The `format` isn't a thing for `number` JSON types, so it should just be removed.

Reply:
Removed and changed to just `'type': ['number']`